Source/Painting/SvgLinearGradientServer.cs

```diff
@@ -78,44 +69,37 @@ public override Brush GetBrush(SvgVisualElement renderingElement, ISvgRenderer r
 
                 using (var transform = EffectiveGradientTransform)
                 {
-                    var midPoint = new PointF((points[0].X + points[1].X) / 2, (points[0].Y + points[1].Y) / 2);
                     transform.Translate(bounds.X, bounds.Y, MatrixOrder.Prepend);
                     if (this.GradientUnits == SvgCoordinateUnits.ObjectBoundingBox)
                     {
                         // Transform a normal (i.e. perpendicular line) according to the transform
                         transform.Scale(bounds.Width, bounds.Height, MatrixOrder.Prepend);
-                        transform.RotateAt(-90.0f, midPoint, MatrixOrder.Prepend);

```

Comment:
I'm not sure what that rotation was supposed to do - can you explain that? Obviously it's not needed...